libr/io/p/io_debug.c

```diff
@@ -211,6 +211,7 @@ static void trace_me(void) {
 	ptrace (PT_SIGEXC, getpid(), NULL, 0);
 #endif
 #else
+	setsid ();
```

Comment:
You are right. For now, the debugee `stdout` still output to the console if that is what you mean by console redirection. I am still testing this out, but I believe this can be removed because I do not see any effect of fixing the bug with this.

Reply:
>For now, the debugee stdout still output to the console if that is what you mean by console redirection
I meant redirecting data into the program's stdin with Cutter

Reply:
I will remove this because I think this is no longer necessary.